crates/rpc/rpc-eth-types/src/error/mod.rs

```diff
@@ -581,6 +581,7 @@ impl From<InvalidTransaction> for RpcInvalidTransactionError {
             InvalidTransaction::Eip1559NotSupported |
             InvalidTransaction::Eip4844NotSupported |
             InvalidTransaction::Eip7702NotSupported => Self::TxTypeNotSupported,
+            _ => todo!(),
```

Comment:
we cant have a todo here

this must be exhaustive and must not panic

Reply:
sure, done